docs/cloud/managing-airbyte-cloud/manage-credits.md

```diff
@@ -24,11 +24,9 @@ To buy credits:
 
     :::
 
-    To buy more credits or discuss a custom plan, reach out to [Sales](https://airbyte.com/talk-to-sales).
+To buy more credits or discuss a custom plan, reach out to [Sales](https://airbyte.com/talk-to-sales).
```

Comment:
The indentation should still be here, so it's not ending the list for this sentence.